### PR TITLE
rerender and update

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
@ocefpaf only you can merge this one. It is needed for others...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,13 +36,17 @@ test:
...
     package is not meant to be deployed for cryptographic purposes for which more comprehensive
-    (and widely tested) libraries such as 'OpenSSL' should be used."
+    (and widely tested) libraries such as 'OpenSSL' should be used.
 
 extra:
   recipe-maintainers:
     - ocefpaf
+    - johanneskoester
+    - bgruening
+    - daler
```
Comment (ocefpaf) [inline recipe/meta.yaml:52]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,14 +19,14 @@ build:
...
-  build:  # [not win32]
-    - r-base  # [not win32]
+requirements:
+  build:
+    - r-base
     - posix  # [win64]
     - m2w64-toolchain  # [win64]
     - gcc  # [not win]
-  run:  # [not win32]
-    - r-base  # [not win32]
```
Comment (ocefpaf) [inline recipe/meta.yaml:29]: This was an old one :smile:

**Comment by ocefpaf** [comment]
@bgruening AppVeyor is quite backlogged due to the new r-recipes added. So I am merging this without waiting for it but let's keep an eye in the CIs.

**Comment by bgruening** [comment]
Thanks a lot!

File: recipe/meta.yaml (hunk 2 above)
Comment (bgruening) [reply recipe/meta.yaml:29]: Indeed, so much progress during this time :)

**Pull request merged**